### ISSUE TITLE
TankDrive.h return in void motorStopped()

### ISSUE THREAD

**Issue by NooclearWessel** [reeltwo/Reeltwo#13]
Hello.
I am getting a compile error in Reeltwo\src\drive\TankDrive.h where motorStopped() is returning from void. Should this be bool?

```
    void motorStopped()
    {
        return fMotorsStopped;
    }
```

**Comment by rimim** [comment]
Yes. But it would only return true if stop() was called. Motors may not be moving. To check if the motors are really not moving you would also have to check that fDriveThrottle and fDriveTurning are 0. Honestly we should probably just get rid of fMotorsStopped and just check fDriveThrottle/fDriveTurning.
